### DISCUSSION TITLE
v0.1.2-beta

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/pogoplusle#7] [Announcements]
## What's Changed
* Add support for Samsung devices by @tyler-pugmire in https://github.com/Mygod/pogoplusle/pull/6

## New Contributors
* @tyler-pugmire made their first contribution in https://github.com/Mygod/pogoplusle/pull/6

**Full Changelog**: https://github.com/Mygod/pogoplusle/compare/v0.1.1...v0.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/pogoplusle/releases/tag/v0.1.2'>v0.1.2-beta</a>.</em>

**Comment by Mygod** [comment]
@tyler-pugmire Feel free to verify that this new version works on your Samsung device. :)

**Reply by tyler-pugmire** [reply]
The new version works on both of my devices.
